src/components/postList/PostList.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+import { Post } from './Post';
+
+interface Props {
+  posts: Posts[];
```

Comment:
почему posts?

Reply:
Не понял что ты имеешь ввиду

Reply:
я же посты маплю, по этому постс

Reply:
posts[] - массив массивов?
```suggestion
  posts: Post[];
```